R/utils.R

```diff
@@ -84,3 +84,14 @@ matchArg <- function(arg, choices) {
 
   return (arg)
 }
+
+#' Set Row Order
+#' 
+#' setroworder reorders the rows of a data.table to the new order provided.
+#' @param a data.table
+#' @param neworder Character vector of the new column name ordering. 
+#' @export
+setroworder <- function(x, neworder) {
```

Comment:
oooooooooh we could maybe use this to reorder the collection data so that it's sent to the client in the right order... 😄

Reply:
i ended up changing my mind about using this in plot.data, but i liked it enough to leave it in utils